src/lib/games/connect-four/ConnectFourBotController.ts

```diff
@@ -0,0 +1,12 @@
+import { cast } from '#utils/util';
+import { ticTacToe } from '@skyra/ai';
+import { BaseBotController } from '../base/BaseBotController';
+import type { ConnectFourGame } from './ConnectFourGame';
+
+export class ConnectFourBotController extends BaseBotController<number> {
+	public await(): number {
+		const game = cast<ConnectFourGame>(this.game);
+		// TODO: Replace this with ConnectFour AI
+		return ticTacToe(game.board);
```

Comment:
TODO

Reply:
Of course it's a TODO, not only the PR is marked as draft, but the latest `@skyra/ai` npm release doesn't have ConnectFour.

Reply:
Doesn't mean I cannot comment it tho :^)